From 9cb8af518c3b11ed8525bc9e409a2720b645cf99 Mon Sep 17 00:00:00 2001
From: Art Limbaga <alimbaga@tektelic.com>
Date: Tue, 28 Jun 2022 13:33:25 -0600
Subject: [PATCH 3/4] gruvbox colorscheme

---
 config.def.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/config.def.h b/config.def.h
index 1edb647..c0439f8 100644
--- a/config.def.h
+++ b/config.def.h
@@ -9,9 +9,9 @@ static const char *fonts[] = {
 static const char *prompt      = NULL;      /* -p  option; prompt to the left of input field */
 static const char *colors[SchemeLast][2] = {
 	/*     fg         bg       */
-	[SchemeNorm] = { "#bbbbbb", "#222222" },
-	[SchemeSel] = { "#eeeeee", "#005577" },
-	[SchemeOut] = { "#000000", "#00ffff" },
+	[SchemeNorm] = { "#ebdbb2", "#282828" },
+	[SchemeSel] = { "#ebdbb2", "#458588" },
+	[SchemeOut] = { "#ebdbb2", "#8ec07c" },
 };
 /* -l option; if nonzero, dmenu uses vertical list with given number of lines */
 static unsigned int lines      = 0;
-- 
2.37.1

